package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fetch-openapi",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
revert this.  handle version bumps in their own commit with `npm version patch`.